pkg/patterns/declarative/reconciler.go

```diff
@@ -403,3 +411,30 @@ func (r *Reconciler) IsKustomizeOptionUsed() bool {
 func (r *Reconciler) SetSink(sink Sink) {
 	r.options.sink = sink
 }
+
+func parseListKind(infos *manifest.Objects) error{
+		for i, item := range infos.Items{
+			if item.Kind == "List" {
+				fmt.Println("haaa!")
+				obj := item.UnstructuredObject()
+
+				err := obj.EachListItem(func(obj runtime.Object) error{
+					castItem := obj.(*unstructured.Unstructured)
```

Comment:
Nit: renaming castItem to `itemUnstructured` may be clearer (similarly, you might rename `obj` to `itemObject`).  (I do like rust's ability to redefine variables with the same name but with a different type, avoids these problems)